.github/workflows/odh-fork-sync.yml

```diff
@@ -1,15 +1,18 @@
 name: Call sync on OpenDataHub CFO fork sync
 on:
   release:
-    types: [published]
+    types:
+      [published]
```

Comment:
Shouldn't that be:
```
types:
  - published
```

Reply:
I am sure it could be but I just followed the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release)

Reply:
I thought it's either:
```
types: [published]
```
Or:
```
types:
  - published
```
But maybe:
```
types:
  [published]
```
Also works. Just that it's a bit uncommon :).

Reply:
All three of those work but to keep things simple Ill set to 
```
types:
  - published
```
As we are only working with 1 type not multiple so theres no need for a list :+1: